src/App.jsx

```diff
@@ -2,8 +2,13 @@ import { Route, NavLink, BrowserRouter as Router, Routes } from 'react-router-do
 import { useFetch } from '../src/hooks/useFetch';
 import './index.css';
 import React from 'react';
+import ProductDetail from './components/ProductDetail';
+import Products from './components/Products';
+
 
 function Home() {
+
+  
   return (
     <div class="homepg">
```

Comment:
Needs to be `className` throughout your application. Your console has multiple errors. Please do not ignore the errors.